lib/Config.js

```diff
@@ -574,9 +574,20 @@ function parseIntegrityChecks(config) {
     return integrityChecks;
 }
 
+const serverAccessLogsModes = {
+    DISABLED: 'disabled',
+    LOG_ONLY: 'log-only',
+    ENABLED: 'enabled',
```

Comment:
```suggestion
    DISABLED: 'DISABLED',
    LOG_ONLY: 'LOG_ONLY',
    ENABLED: 'ENABLED',
```

I'd prefer we keep same value and key for enums

Reply:
Done.